requirements/test-requirements.txt

```diff
@@ -17,9 +17,6 @@ kaleido
 # Required by tuning tests
 hyperopt
 # Required by recipes tests
-# Pinned because ipython>=8.13 doesn't support Python 3.8,
-# which is MLflow's minimum supported Python version
-ipython<8.13.0
```

Comment:
MLflow does support Python 3.8 now. We can remove this.

Reply:
You mean ipython supports python 3.8 right?

Reply:
I think I misread this comment :) and am confused now.

Reply:
I see, ipython>=8.13 supports Python >= 3.9.

Reply:
`Pinned because ipython>=8.13 doesn't support Python 3.8`. We remove this because ipython>=8.13 supports python 3.8 now?

Reply:
That also works 👍

Reply:
Let's see what happens with this.